### ISSUE TITLE
0.7.1 Weird linting output

### ISSUE THREAD

**Issue by urbany** [rwjblue/ember-cli-template-lint#273]
Just updated from 0.7.0 where everything was fine to 0.7.1 and now when I start `ember serve` the linting output is very weird, full of new lines:

![lint-output](https://user-images.githubusercontent.com/50884/28715253-f68e76fa-738e-11e7-87cf-66fb30b26404.png)

Also if I run: `./node_modules/.bin/ember-template-lint app/` and `./node_modules/.bin/ember-template-lint lib/`I get no errors returned. Which I think is correct. So not only the output is weird there should be no errors found when doing `ember serve`

Using: ember-cli@2.14.1

I tried deleting node_modules and a clean `npm i` and now everytime I start ember serve it looks like all my templates are printed:
```
my-project/templates/template-name-1.hbs


my-project/templates/template-name-2.hbs


my-project/templates/template-name-3.hbs


my-project/templates/template-name-4.hbs


my-project/templates/template-name-5.hbs


my-project/templates/template-name-6.hbs

...

===== 192 Template Linting Errors
```

Thanks in advance!

**Comment by kpfefferle** [comment]
I am getting a similar output when running `ember test` using `0.7.1` on our project:

```
⠸ BuildingWarning: ignoring input sourcemap for vendor/ember/ember.debug.js because ENOENT: no such file or directory, open '/Users/kevin/code/company/project/tmp/source_map_concat-input_base_path-NzY6LFz4.tmp/vendor/ember/ember.debug.map'






































































===== 69 Template Linting Errors

Warning: ignoring input sourcemap for vendor/ember/ember-testing.js because ENOENT: no such file or directory, open '/Users/kevin/code/company/project/tmp/source_map_concat-input_base_path-PkUYiGsZ.tmp/vendor/ember/ember-testing.map'
cleaning up...
Built project successfully. Stored in "/Users/kevin/code/company/project/tmp/class-tests_dist-X34Ntwfq.tmp".
```

**Comment by rwjblue** [comment]
Yes, the issue is that we are including the extra newlines regardless of errors, and then it ends up looking pretty odd.

**Comment by rwjblue** [comment]
I would love help digging into this, as I'm fairly constrained for time ATM.  The 0.7.1 release updates to leverage the `errorsToMessages` code in the linter itself (instead of duplicating the presentation logic both here and in ember-template-lint).  That method is [here](https://github.com/rwjblue/ember-template-lint/blob/6e65e7bcf4885dc8f633ad77a9c6053c82ce3e66/lib/index.js#L151-L179), and its tests are [here](https://github.com/rwjblue/ember-template-lint/blob/6e65e7bcf4885dc8f633ad77a9c6053c82ce3e66/test/acceptance-test.js#L520-L571).

**Comment by Exelord** [comment]
OMW!

**Comment by Exelord** [comment]
I think I fixed an issue related to printing file paths in https://github.com/rwjblue/ember-template-lint/pull/250

But... There still seems to be a bug in `ember-cli-template-lint` because errors are not correctly listed. :/

**Comment by chriskrycho** [comment]
Saw this just now, and extra weirdly running the linter directly on the listed files says they have no errors.

**Comment by rwjblue** [comment]
OK, ember-template-lint@0.8.1 was just released with the fixes from https://github.com/rwjblue/ember-template-lint/pull/250.  Can folks update (might need to `yarn upgrade`) and confirm things are working properly>?

**Comment by Exelord** [comment]
For me, it's still bugged :/
As the `ember-template-list` seems to be fixed already I think there I still some errors related to: https://github.com/rwjblue/ember-cli-template-lint/pull/272/files

I can try to debug it in the next week if it will stay untouched

**Comment by rwjblue** [comment]
Thank you @Exelord! I believe this is now fixed and released in v0.7.3.

Closing, but happy to reopen if others have issues still...
